web/package.json

```diff
@@ -86,8 +86,7 @@
     "redux": "^3.7.2",
     "redux-thunk": "^2.3.0",
     "tippy.js": "^2.6.0",
-    "updeep": "^1.0.0",
-    "yup": "^0.25.1"
```

Comment:
This library has a funky dependency on an old version of the Babel runtime and it was causing build problems.  We only use it in our validating input components, which we aren't using except in one place and we shouldn't be.  So I removed those too...